ToDoList-Redux/Binders/ReduxFramework.swift

```diff
@@ -83,47 +101,28 @@ extension World {
     )
 }
 
-
-// MARK: - PRISM
-
-extension AppAction {
-    public var task: TaskList.Action? {
-        get {
-            guard case let .task(value) = self else { return nil }
-            return value
-        }
-        set {
-            guard case .task = self, let newValue = newValue else { return }
-            self = .task(newValue)
-        }
-    }
-}
-
-
 // MARK: - EXTENSIONS
 
 extension TaskList {
-    
     static func viewModel<S: StoreType>(store: S) -> ObservableViewModel<TaskList.Action, TaskList.State>
-    where S.ActionType == AppAction, S.StateType == AppState {
+    where S.ActionType == TaskAction, S.StateType == [Task] {
```

Comment:
It could be AppAction/AppState here. But if you plan to eventually move this view to another module, you can use strictly the essential to build a view and handle its actions, ignoring things like AppLifecycle for example. This is totally optional.